### PR TITLE
Cphi

### PR DESCRIPTION
**Pull request by Ravoxsg**
Computing the gradient via RELAX estimator

### CODE REVIEW

File: annex_network.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.autograd import Variable
+
+
+
+
+class AnnexNetwork(nn.Module):
```
Comment (raeidsaqur) [inline annex_network.py:16]: I think 'ControlVariate(nn.Module)' is more clear for the class name.

**Review by raeidsaqur — APPROVED** [review]
LGTM

**Review by raeidsaqur — APPROVED** [review]
